Autocoders/Python/src/fprime_ac/utils/XmlParser.py

```diff
@@ -207,7 +207,7 @@ def parse_file(self, filename):
                 # requirement for parsing a second file is that it has a same
                 # named root Element.
 
-                if self.__root is not None and root_name_checked == False:
+                if self.__root is not None and root_name_checked:
```

Comment:
This changed logic, I think.  Seems it should read:

```
if self.__root is not None and not root_name_checked:
```

Reply:
Fixed.